src/components/PageSections/Menu/ThemesNav.js

```diff
@@ -2,11 +2,19 @@ import React from 'react';
 import { injectIntl, useIntl } from 'react-intl';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { search as i18n } from 'js/i18n/definitions';
+import { misc as i18n } from 'js/i18n/definitions';
+import ExternalLink from 'components/ExternalLink';
 
 const ThemesNav = props => {
 
   const intl = useIntl();
+  const lang = intl.locale
+
+  if (lang === 'en') {
+    props.navigation.map( theme => {
+      theme.text = theme.text.split('and').join('&')
+    })
+  }
```

Comment:
I wonder if this should be managed by content instead of dev. Do they only want the ampersand to be displayed in this part of the UI?

Reply:
Chase wanted to change it so that we could get a little horizontal space back. The Themes Nav is operating on really tight character count and this gave it a little more to work with.